parm/config/gfs/config.wave

```diff
@@ -99,18 +99,18 @@ fi
 export WAVHINDH=${WAVHINDH:-0}
 export FHMIN_WAV=${FHMIN_WAV:-0}
 export FHOUT_WAV=${FHOUT_WAV:-3}
-export FHMAX_HF_WAV=${FHMAX_HF_WAV:-120}
+export FHMAX_HF_WAV=${FHMAX_HF_WAV:-0} # 120 for ops
 export FHOUT_HF_WAV=${FHOUT_HF_WAV:-1}
 export FHMAX_WAV_IBP=180
 if (( FHMAX_WAV < FHMAX_WAV_IBP )); then export FHMAX_WAV_IBP=${FHMAX_GFS} ; fi
 
 # gridded and point output rate
 export DTFLD_WAV=$(( FHOUT_HF_WAV * 3600 ))
-export DTPNT_WAV=${DTPNT_WAV:-3600}
+export DTPNT_WAV=${DTPNT_WAV:-10800} # 3600 for ops
 export FHINCP_WAV=$(( DTPNT_WAV / 3600 ))
 
 # Selected output parameters (gridded)
-export OUTPARS_WAV=${OUTPARS_WAV:-"WND HS FP DP PHS PTP PDIR"}
+export OUTPARS_WAV=${OUTPARS_WAV:-"WND CUR ICE HS T01 T02 DIR FP DP PHS PTP PDIR CHA"}
```

Comment:
Same question as in `config.base` for `FHOUT_GFS`.
Infact, there are several variables in here that are defined as `var=${var:-default_value}`.  Do they need to be this way now that `config.defaults.s2sw` is no longer being sourced to provide those?

Reply:
Probably not. Can remove if you'd like.

Reply:
Updated.